tests/integration/targets/junos_command/tests/netconf_text/no_config.yaml

```
@@ -0,0 +1,19 @@
+---
+- debug: msg="START netconf_text/no_config.yaml on connection={{ ansible_connection
+    }}"
+
+- name: test bad condition
+  register: result
+  ignore_errors: true
+  junipernetworks.junos.junos_command:
+    commands:
+      - show configuration system ntp
+    display: text
+    provider: '{{ netconf }}'
+
+- assert:
+    that:
+      - result.failed == false
```

I _think_ you can also check the output here?
```suggestion
      - result.failed == false
      - result.stdout == [""]
```